Changelogs for Digital Twin Manager UI 1021.1.4 (#3246)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1021-3-0-fixed-unresponsive-add-asset-and-missing-import-asset-buttons-in-dtm.md

---

> date: "2025-04-23"

Reviewer: ```suggestion
date: 
```

Reviewer: The date will be added automatically on deployment.

> title: "Fixed unresponsive 'Add Asset' and missing 'Import Asset' buttons in DTM."

Reviewer: ```suggestion
title: "Add asset button and Import asset button now work properly"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2012"
version: "1021.3.0"
---

> Previously, the 'Add Asset' button was unresponsive and the 'Import Asset' button was missing when the Asset Navigator was installed in the Digital Twin Manager. These issues have now been resolved.

Reviewer: ```suggestion
Previously,  when the asset navigator was installed in the Digital Twin Manager, the **Add asset** button was unresponsive and the **Import asset** button was missing. These issues have now been resolved and both buttons work properly.
```

> Previously,  when the asset navigator was installed in the Digital Twin Manager, the **Add asset** button was unresponsive and the **Import asset** button was missing. These issues have now been resolved and both buttons work properly.

Reviewer: ```suggestion
Previously, when the asset navigator was installed in the Digital Twin Manager, the **Add asset** button was unresponsive and the **Import asset** button was missing. These issues have now been resolved and both buttons work properly.
```

---
date: ""
title: "Fixed unresponsive 'Add Asset' and missing 'Import Asset' buttons in DTM."
product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2012"
version: "1021.3.0"
---
* Fix for CTM-2012 and CTM-2009

* Fix for linting error

* Fix for e2e tests

* Updated the testcase number

> * Added an additional test case to cover on Add group button.

Reviewer 2: @spau6 , please update the right content for the changelog here

Reviewer 3: Updated
